src/handlers/bet.rs

```diff
@@ -130,3 +160,52 @@ async fn create_new_odds(
         .await
         .unwrap()
 }
+
+pub async fn get_bet_history_by_user_id(
+    mut bet_repository: BetRepository,
+    mut match_repository: GameMatchRepository,
+    user_id: Uuid,
+) -> Vec<Bet> {
+    let user_bet_history = bet_repository
+        .read_many(&BetGetByUserId { user_id })
+        .await
+        .unwrap();
+
+    let mut bet_history = Vec::new();
+
+    for bet in user_bet_history {
+        let game_match = match_repository
+            .read_one(&GameMatchGetById {
+                id: bet.game_match_id,
+            })
+            .await
+            .unwrap();
+
+        let predicted_team = match bet.expected_outcome {
+            GameMatchOutcome::WinA => game_match.name_a.clone(),
+            _ => game_match.name_b.clone(),
+        };
+
+        let bet_date = format!(
+            "{}/{}/{} - {}:{}",
+            bet.created_at.day(),
+            bet.created_at.month(),
+            bet.created_at.year(),
+            bet.created_at.hour(),
+            bet.created_at.minute()
+        );
```

Comment:
Přidej prosím, aby se před hodiny a minuty doplnila nula pokud to je jednociferné číslo